### PR TITLE
feat(lang): add Meson language support

### PR DESCRIPTION
**Pull request by unrelentingtech**
https://mesonbuild.com/Syntax.html
https://github.com/bearcove/tree-sitter-meson

Screen_2022-04-29-00:44:17

### CODE REVIEW

File: runtime/queries/meson/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,62 @@
+(string_literal) @string
+
+(boolean_literal) @constant.builtin.boolean
+(integer_literal) @constant.builtin.numeric.integer
+
+(comment) @comment.line
+(function_id) @function
+(keyword_arg_key) @property
```
Comment (the-mikedavis) [inline runtime/queries/meson/highlights.scm:8]:
```suggestion
(keyword_arg_key) @variable.other.member
```

File: runtime/queries/meson/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,62 @@
+(string_literal) @string
+
+(boolean_literal) @constant.builtin.boolean
+(integer_literal) @constant.builtin.numeric.integer
```
Comment (the-mikedavis) [inline runtime/queries/meson/highlights.scm:4]:
```suggestion
(integer_literal) @constant.numeric.integer
```

**Review by the-mikedavis — CHANGES_REQUESTED** [review]
Couple of minor things but looks good otherwise 👍 

Let's await https://github.com/bearcove/tree-sitter-meson/issues/3 too

**Review by archseer — APPROVED** [review]
Looks good, but can you resolve the merge conflict?

**Comment by unrelentingtech** [comment]
rebased again

**Pull request merged**